content/applications/sales/crm/track_leads/marketing_attribution.rst

```diff
@@ -0,0 +1,142 @@
+=============================
+Marketing attribution reports
+=============================
+
+Use the Odoo *CRM* app to compile a *marketing attribution report*, which analyzes the source of
+leads in the CRM, and groups them in such a way as to calculate marketing's overall impact on lead
+generation, attribution, won rate, and more.
+
+Leads Analysis dashboard
+========================
+
+Begin by navigating to the :guilabel:`Leads Analysis` dashboard by going to :menuselection:`CRM app
+--> Reporting --> Leads`.
+
+.. tip::
+   Alternatively, reports can also be run on the :menuselection:`CRM app --> Leads` dashboard or the
+   :menuselection:`CRM app --> Sales --> My Pipeline` dashboard (in case the :guilabel:`Leads`
+   dashboard is not available); both dashboards contain the necessary :guilabel:`Filters` and
+   :guilabel:`Group By` criteria to run an attribution report.
+
+.. seealso::
+   - :doc:`Convert leads into opportunities <../acquire_leads/convert>`
+   - :doc:`Create leads <../acquire_leads/email_manual>`
+
+.. image:: marketing_attribution/reporting-tab-and-leads.png
+   :align: center
+   :alt: Open the CRM app and click on the Reporting tab at the top of the page, then click Leads.
+
+The :icon:`fa-area-chart` :guilabel:`(Graph)` view will be shown by default, with :guilabel:`Active
+or Inactive` and :guilabel:`Created on: YYYY` filters active in the :guilabel:`Search...` bar. The
+graph visualization displays the number of leads generated, by month and by sales team, with each
+sales team attributed to its own color per bar (month) shown.
+
+Switch the view to the :icon:`oi-view-list` :guilabel:`(List)` option, by clicking the respective
```

Comment:
```suggestion
Switch the view to the :icon:`oi-view-list` :guilabel:`(list)` option, by clicking the respective
```
It's my understanding that we are only meant to capitalize Kanban, when it comes to views... but a quick double-check with SALI about this may be best